Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/AdminLogsController.cs

```diff
@@ -386,12 +386,16 @@ public HttpResponseMessage AddLogSetting([FromBody] UpdateLogSettingsRequest req
         {
             try
             {
-                var isAdmin = this.UserInfo.Roles.Contains(this.PortalSettings.AdministratorRoleName);
-                if (isAdmin)
-                {
-                    return this.Request.CreateResponse(HttpStatusCode.Unauthorized);
+                if (!this.UserInfo.IsSuperUser)
+                {
+                    var isAdmin = this.UserInfo.Roles.Contains(this.PortalSettings.AdministratorRoleName);
+                    if (isAdmin)
+                    {
+                        return this.Request.CreateResponse(HttpStatusCode.Unauthorized);
+                    }
+
+                    request.LogTypePortalID = this.PortalId.ToString();
```

Comment:
I think we just want to completely deny access unless you're a superuser.  I don't think there's a point in checking against the administrator role anymore.

```suggestion
                    return this.Request.CreateResponse(HttpStatusCode.Unauthorized);
```